### PR TITLE
Video demo is unsupported on simulator

### PR DESCRIPTION
**Pull request by sigurdm**
Display a message stating this.

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/video_demo.dart [hunk 1]
```diff
@@ -321,6 +329,7 @@ class _VideoDemoState extends State<VideoDemo>
 
   final GlobalKey<ScaffoldState> scaffoldKey = new GlobalKey<ScaffoldState>();
   final Completer<Null> connectedCompleter = new Completer<Null>();
+  bool unsupported = false;
```
Comment (mravn-google) [inline examples/flutter_gallery/lib/demo/video_demo.dart]: Could be inverted to avoid a negative boolean variable. `isSupported`?

**Review by mravn-google — APPROVED** [review]
LGTM

**Comment by eseidelGoogle** [comment]
Do we know why?  Is this temporary?

FYI @HansMuller @sethladd

File: examples/flutter_gallery/lib/demo/video_demo.dart (hunk 1 above)
Comment (sigurdm) [reply examples/flutter_gallery/lib/demo/video_demo.dart]: done

**Comment by sigurdm** [comment]
We are using CVOpenGLESTextureCacheCreate to handle textures on iOS that does not work on the simulator.
Looking forward we should be able to find an alternative.

**Comment by lishuhao** [comment]
If this plugin dose not work on simulator,we should declare that

**Comment by lishuhao** [comment]
@sigurdm

**Pull request merged**